hw/ip/clkmgr/dv/cov/clkmgr_manual_cov_unr_excl.el

```diff
@@ -0,0 +1,49 @@
+// Copyright lowRISC contributors.
+// Licensed under the Apache License, Version 2.0, see LICENSE for details.
+// SPDX-License-Identifier: Apache-2.0
+
+// This excludes toggle coverage for output ports driven by constant and inputs
+// tied to constants.
+//==================================================
+// This file contains the Excluded objects
+// Generated By User: maturana
+// Format Version: 2
+// Date: Thu Aug 11 12:43:08 2022
+// ExclMode: default
+//==================================================
+CHECKSUM: "2617278896 3012009448"
+INSTANCE: tb.dut
+ANNOTATION: "[UNR] This is driven by a constant."
+Toggle cg_en_o.aon_infra "logic cg_en_o.aon_infra[3:0]"
```

Comment:
are these design outputs? If so, they should be caught by UNR tool?

Reply:
I agree, they should, but I have run cov-unr multiple times, and it just doesn't detect these obvious ones.